### PR TITLE
Get All Likes For a Specific Post

### PR DESCRIPTION
**Pull request by janko101**
# Changes proposed in this pull request
- Adds get request that  returns all likes for a given post.
- Sends user email with the like

### CODE REVIEW

File: src/main/java/se/kth/sda/skeleton/postlikes/PostLikeService.java [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 import se.kth.sda.skeleton.auth.AuthService;
+import se.kth.sda.skeleton.comments.Comment;
```
Comment (eeels22) [inline src/main/java/se/kth/sda/skeleton/postlikes/PostLikeService.java]: Is this import used?

File: src/main/java/se/kth/sda/skeleton/postlikes/PostLikeController.java [hunk 2]
```diff
@@ -4,6 +4,9 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
+import se.kth.sda.skeleton.comments.Comment;
```
Comment (eeels22) [inline src/main/java/se/kth/sda/skeleton/postlikes/PostLikeController.java]: Is the Comment import used?

File: src/main/java/se/kth/sda/skeleton/postlikes/PostLikeService.java (hunk 1 above)
Comment (janko101) [reply src/main/java/se/kth/sda/skeleton/postlikes/PostLikeService.java]: Nope...good spot

File: src/main/java/se/kth/sda/skeleton/postlikes/PostLikeController.java (hunk 2 above)
Comment (janko101) [reply src/main/java/se/kth/sda/skeleton/postlikes/PostLikeController.java]: Removed that one too

**Review by eeels22 — APPROVED** [review]
Great, thanks Janko!

**Review by NeuralAlchemist — APPROVED** [review]
looks good!

**Pull request merged**